src/main/java/org/opensearch/neuralsearch/processor/TextEmbeddingProcessor.java

```diff
@@ -26,18 +32,31 @@ public final class TextEmbeddingProcessor extends InferenceProcessor {
 
     public static final String TYPE = "text_embedding";
     public static final String LIST_TYPE_NESTED_MAP_KEY = "knn";
+    public static final String SKIP_EXISTING = "skip_existing";
+    public static final boolean DEFAULT_SKIP_EXISTING = Boolean.FALSE;
```

Comment:
```suggestion
    public static final boolean DEFAULT_SKIP_EXISTING = false;
```

You are using primitive datatype so false should be fine.